docker-compose.yml

```diff
@@ -37,8 +37,12 @@ services:
     env_file: .env
   frontend:
     container_name: frontend
+    volumes:
+      - './app:/usr/app'
+      - '/usr/app/.next'
+      - '/usr/app/node_modules'
```

Comment:
Why are you mounting a volume to `/usr/app/node_modules` that is available on the host? This would just be available at `./app/node_mdoules` by the volume on line 40.